test/wantmanager/index.spec.js

```diff
@@ -63,14 +63,16 @@ describe('WantManager', () => {
         expect(calls.connects).to.have.length(6)
         expect(calls.messages).to.have.length(6)
 
-        calls.messages.forEach((m, i) => {
-          expect(m[0]).to.be.eql(calls.connects[i])
+        for (let i = 0; i < calls.messages.length; i++) {
+          const m = calls.messages[i]
```

Comment:
Just a note (no need to change it). I generally try not to use single letter variables (for easier grepping). A nice trick a colleague showed me as to simply double the letter if you want a single letter variable. So it would be `ii` and `mm`.